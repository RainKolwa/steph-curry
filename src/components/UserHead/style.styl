.user-head
  display flex
  flex-direction column
  align-items center
  img
    width 80px
    height 80px
    border-radius 40px