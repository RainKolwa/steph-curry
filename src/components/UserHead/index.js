import React from 'react'
import PropTypes from 'prop-types'
import './style.styl'

const UserHead = ({ user }) => {
  const { name, avatar } = user
  return (
    <div className="user-head">
      <img src={avatar} alt={name} />
      <span>{name}</span>
    </div>
  )
}

UserHead.propTypes = {
  user: PropTypes.shape({
    name: PropTypes.string.isRequired,
    avatar: PropTypes.string.isRequired,
  }).isRequired,
}

export default UserHead
