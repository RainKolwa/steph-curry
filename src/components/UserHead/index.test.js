import React from 'react'
import renderer from 'react-test-renderer'
import UserHead from './index'

it('renders correctly', () => {
  const user = {
    name: 'Foo',
    avatar: 'Bar',
  }
  const userhead = renderer.create(<UserHead user={user} />).toJSON()
  expect(userhead).toMatchSnapshot()
})
