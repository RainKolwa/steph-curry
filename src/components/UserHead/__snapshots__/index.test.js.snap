exports[`test renders correctly 1`] = `
<div
  className="user-head">
  <img
    alt="Foo"
    src="Bar" />
  <span>
    Foo
  </span>
</div>
`;
